packages/cloud-functions/.gitignore

```diff
@@ -0,0 +1,12 @@
+dist/
+
+# Firebase cache
+.firebase/
+
+# Only check in encrypted .env file
+.env
+.env.alfajores
```

Comment:
Glad you're giving this package its own .gitignore! I added `.env.alfajores` and `.env.mainnet` to the root gitignore...can you please delete those from there? This is definitely the right way to do it :)